### DISCUSSION TITLE
pulseeffects not effecting audio in plain x session

### DISCUSSION THREAD

**Discussion by jtharkey** [wwmm/easyeffects#1636] [General]
i'm still running pulseeffects, in ubuntu 21.10. i need to run it in a plain x session. i start pulseeffects, then run my app and send the audio to its sink, and i hear audio like expected. except the effects don't do anything. using gsettings i have verified the effects' state are true

if i run it on the same os install, but in xfce, the same effects/settings do work. the two effects i'm using are eq and filter

i can't see anything that is obviously different between the two ways i'm running things. any idea why this would be

**Comment by wwmm** [comment]
Which PulseEffects version are you using? Depending on how old it is you may be seeing a bug related to `dconf`. Older PulseEffects versions do not interact well with recent dconf versions. It would also help to see some logs. Take a look at the `Basic Logs` section of https://github.com/wwmm/easyeffects/wiki/Reporting-Bugs. I think that in your case there is no need for Pavucontrol images. Just the EasyEffects logs should be enough.

**Comment by jtharkey** [comment]
pulseeffects is at version 4.8.4-1. i've read about the dconf issues. right now i have dconf version 0.40.0-1, however, i've also tried version 0.38.0-2. here is the log with it running dconf 0.40.0-1

[log.txt](https://github.com/wwmm/easyeffects/files/9077773/log.txt)

**Reply by wwmm** [reply]
The log seems fine. But it is still a good idea to upgrade to the latest bugfix release `4.8.7`. Maybe something different happens.

**Comment by jtharkey** [comment]
i will try that, thank you!
